downstream/modules/platform/con-upgrade-planning.adoc

```diff
@@ -18,10 +18,11 @@ The `context` attribute enables module reuse. Every module ID includes {context}
 = Upgrade planning
 
 [role="_abstract"]
-This section covers changes that you should keep in mind as you attempt to upgrade your {ControllerName} instance
+Depending on your installation, there may be several changes to consider as you attempt to upgrade your {PlatformName}.
 
+.Upgrading {ControllerName}
+If you plan on upgrading your {ControllerName} instance along with your {PlatformNameShort}, review the following information:
 
-- Even if you already have a valid license from a previous version, you must still provide your credentials or a subscriptions manifest again upon upgrading to {ControllerName} 3.8. See link:https://docs.ansible.com/automation-controller/latest/html/userguide/import_license.html[Import a subscription] in the Automation Controller User Guide.
-- If you need to upgrade Red Hat Enterprise Linux and {ControllerName}, you will need to do a backup and restore of your {ControllerName} data. Refer to link:https://docs.ansible.com/ansible-tower/3.8.3/html/installandreference/upgrade_tower.html#ir-upgrade-existing[Upgrading an Existing Controller Installation] for further detail.
-- Clustered upgrades require special attention to instance and instance groups prior to starting the upgrade. Refer to the link:https://docs.ansible.com/ansible-tower/3.8.3/html/quickinstall/install_script.html#setting-up-the-inventory-file[Setting up the inventory file] and see link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/clustering.html#clustering[Clustering] for details.
-- Prior versions of Ansible Tower used the variable name `rabbitmq_host` during installation. If you are upgrading from a previous version of Tower, and you previously specified `rabbitmq_host` in your inventory, simply rename `rabbitmq_host` to `routable_hostname` before upgrading. See link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/clustering.html#clustering[Clustering] for details.
+- Even if you already have a valid license from a previous version, you must still provide your credentials or a subscriptions manifest again upon upgrading to the latest version of {ControllerName}. See link:https://docs.ansible.com/automation-controller/latest/html/userguide/import_license.html[Import a subscription] in the Automation Controller User Guide.
```

Comment:
Suggestion: `you must still provide` > `you still must provide`

Suggestion: Remove the word `again` since it is redundant of the word `still`. However, that may be a preference thing and I leave that up to you.